content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/csi-node-expand-secret.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CSINodeExpandSecret
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+<!--
+Enable passing secret authentication data to a CSI driver for use
+ during a `NodeExpandVolume` CSI operation.
+-->
+允许在 `NodeExpandVolume` CSI 操作期间将 Secret
```

Comment:
The English original is talking about data that is secret, rather than the Secret API.
Maybe this matches the current text and we can revise it in a different PR; I haven't checked.